upt_macports/templates/hackage.Portfile

```diff
@@ -0,0 +1,19 @@
+{% extends 'base.Portfile' %}
+
+{% block portgroup %}
+PortGroup           haskell 1.0
+{% endblock %}
+{% block nameversion %}
+haskell.setup       {{ pkg._pkgname() }} {{ pkg.upt_pkg.version }}
+revision            0
+{% endblock %}
+
+{% block dist_info %}
+homepage            {{ pkg.upt_pkg.homepage }}
+
+{% endblock %}
+
+{% block versions %}
+
+{{ depends('lib', pkg.upt_pkg.requirements.run) -}}
```

Comment:
You'll need to remove the ```-``` here at the end of this line. Otherwise you'll get ```lint``` errors for every Portfile saying: ```Warning: Line ??? has missing newline (at end of file)```.

Also please set the permissions on this file correctly it should be ```644``` instead of ```755```